app/models/place.rb

```diff
@@ -34,13 +34,17 @@ def update_address(place, raw_place)
   end
 
   def set_slug
-    self.slug = name.parameterize
+    self.slug = name.parameterize + place_id
+  end
+  
+  def self.for_id(place_id)
+    @client = GooglePlaces::Client.new(ENV['GOOGLE_KEY'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.